src/Tizen.Log/Tizen/Log.cs

```diff
@@ -112,17 +114,52 @@ public static void Fatal(string tag, string message, [CallerFilePath] string fil
             Print(Interop.Dlog.LogPriority.DLOG_FATAL, tag, message, file, func, line);
         }
 
-        static void Print(Interop.Dlog.LogPriority priority, string tag, string message, string file, string func, int line)
+        static unsafe void Print(Interop.Dlog.LogPriority priority, string tag, string message, string file, string func, int line)
         {
-            if (String.IsNullOrEmpty(file))
-            {
-                Interop.Dlog.Print(priority, tag, "%s", message);
-            }
-            else
+            int length = tag.Length;
+            Span<byte> tagBuffer = length < 1024 ? stackalloc byte[length + 1] : new byte[length + 1];
+
+            length = message.Length;
+            Span<byte> messageBuffer = length < 1024 ? stackalloc byte[length + 1] : new byte[length + 1];
+
+            fixed (char* charsTag = &MemoryMarshal.GetReference(tag.AsSpan()))
+            fixed (char* charsMessage = &MemoryMarshal.GetReference(message.AsSpan()))
+            fixed (byte* bytesTag = &MemoryMarshal.GetReference(tagBuffer))
+            fixed (byte* bytesMessage = &MemoryMarshal.GetReference(messageBuffer))
             {
-                int index = file.LastIndexOfAny(sep);
-                string filename = file.Substring(index + 1);
-                Interop.Dlog.Print(priority, tag, "%s: %s(%d) > %s", filename, func, line, message);
+                Encoding.ASCII.GetBytes(charsTag, tag.Length, bytesTag, tag.Length);
```

Comment:
Why not UTF8? 
Is Your code working with Korean?
```suggestion
                Encoding.UTF8.GetBytes(charsTag, tag.Length, bytesTag, tag.Length);
```

Reply:
As far as I know, underlying native dlog functions are only working with English(ASCII). But it seems good to use UTF8 to be compatible with another language too.

Reply:
I was wrong. The dlog works well with other languages too. So it should have been UTF8 like you suggested.